app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/provision/cloning.rb

```diff
@@ -49,6 +45,10 @@ def prepare_for_clone_task
     attached_networks.concat(phase_context[:new_networks]).compact!
     specs['networks'] = attached_networks
 
+    # TODO: support multiple values
+    ip_addr = get_option_last(:ip_addr)
+    specs['networks'][0]['ipAddress'] = ip_addr unless !ip_addr || ip_addr.strip.blank?
```

Comment:
Realize this is just a move, but pairing `unless` with `!` and a second boolean condition always causes my brain throws a fault

Could you achieve the same thing with
```suggestion
    specs['networks'][0]['ipAddress'] = ip_addr if ip_addr.present?
```

Reply:
> always causes my brain throws a fault

😆. Fixed